holoviews/plotting/mpl/plot.py

```diff
@@ -354,9 +352,12 @@ def _create_subplots(self, layout, axis, ranges, create_axes):
             # Create axes
             kwargs = {}
             if create_axes:
-                threed = issubclass(vtype, Element3D)
-                subax = plt.subplot(self._layoutspec[r, c],
-                                    projection='3d' if threed else None)
+                opts = self._deep_options(view, 'plot', ['projection'], [CompositeOverlay])
```

Comment:
I think ``_deep_options`` is a useful method but I'm not really happy with the name.